services/provider/server/server.go

```diff
@@ -915,6 +936,17 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 				kernelMountOptions[parts[0]] = parts[1]
 			}
 
+			// SID for RamenDR
+			storageID := util.CalculateMD5Hash(struct {
```

Comment:
storageID has to be within allowed string length. How is that ensured?

Reply:
The CalculateMD5Hash returns a hash that is 32 characters long